spec/meilisearch/index/settings_spec.rb

```diff
@@ -22,7 +22,8 @@
       'stopWords',
       'synonyms',
       'filterableAttributes',
-      'sortableAttributes'
+      'sortableAttributes',
+      'typoTolerance'
```

Comment:
Oh yes, it is! 

The assertion is breaking because I need to match all the present keys in the response, and now we have this new key called `typoTolerance`, it does not mean that we will support them right now. 
The upsides and downsides of having e2e testing 👯